{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -1,5 +1,6 @@
 {% raw %}{% load static i18n {% endraw %}{% if cookiecutter.use_compressor == "y" %}compress{% endif %}{% raw %}%}<!DOCTYPE html>
-<html lang="en">
+{% get_current_language as LANGUAGE_CODE %}
+<html lang="{{ LANGUAGE_CODE }}">
```

Comment:
With this, I'm getting a different value of `en-gb`. This seems to go against [the W3C recommendations](https://www.w3.org/International/articles/language-tags/):

> The golden rule when creating language tags is to keep the tag as short as possible. Avoid region, script or other subtags except where they add useful distinguishing information.

Reply:
I did a bit more reserch, this is fine. It's certainly more correct that always being "en".